test/test-utils.js

```diff
@@ -87,6 +87,12 @@ export const TEST_CHAINS = {
     voucherSubgraphURL: DRONE
       ? 'http://gaphnode:8000/subgraphs/name/bellecour/iexec-voucher'
       : 'http://localhost:8000/subgraphs/name/bellecour/iexec-voucher',
+    debugWorkerpoolOwnerWallet: new Wallet(
+      '0x2c906d4022cace2b3ee6c8b596564c26c4dcadddf1e949b769bcb0ad75c40c33',
+    ),
+    prodWorkerpoolOwnerWallet: new Wallet(
+      '0x2c906d4022cace2b3ee6c8b596564c26c4dcadddf1e949b769bcb0ad75c40c33',
+    ),
```

Comment:
use some other randomly generated wallets (the wallet is used for the voucherMangerWallet)